packages/statemanager/src/proofs/merkle.ts

```diff
@@ -240,7 +240,7 @@ export async function verifyMerkleStateProof(
     const storageProof = stProof.proof.map((value: PrefixedHexString) => hexToBytes(value))
     const storageValue = setLengthLeft(hexToBytes(stProof.value), 32)
     const storageKey = hexToBytes(stProof.key)
-    const proofValue = await verifyMPTProof(storageKey, storageProof, {
+    const proofValue = await verifyMerkleProof(storageKey, storageProof, {
       useKeyHashing: true,
     })
     const reportedValue = setLengthLeft(
```

Comment:
Path related comment (so not for this file): can we align the proof directories from mpt and statemanager, and so (my suggestion) call this one here also:

- `statemanager/src/proofs` -> `statemanager/src/proof`

(we have a lot of stuff in singular - e.g. also `test` (instead of `tests`), so this feels somewhat more as our "convention" here)

Reply:
Looking at the paths from mpt, it seems like we can actually do without  `src/proof` and `src/db`. 

- Proof only exports two files
- DB only exports one

So I moved them over to the src directory (renamed range.ts to rangeProof.ts). This flattens the repo structure and has the benefit of limiting the amount of nested index files.